website2/src/app/clean-air-forum/glossary/page.tsx

```
@@ -1,36 +1,86 @@
+      {/* Clear Air Glossary Section */}
+      <div className="flex flex-col md:flex-row md:space-x-8">
+        {/* Left column: Heading */}
+        <div className="md:w-1/3 mb-4 md:mb-0">
+          <h2 className="text-2xl font-bold text-gray-900">
+            Clear Air Glossary
+          </h2>
         </div>
+        {/* Right column: Glossary content */}
+        <div
+          className="md:w-2/3 space-y-4"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)